redis/commands/timeseries/__init__.py

```diff
@@ -34,22 +34,22 @@ def __init__(self, client=None, **kwargs):
         """Create a new RedisTimeSeries client."""
         # Set the module commands' callbacks
         self.MODULE_CALLBACKS = {
-            CREATE_CMD: bool_ok,
             ALTER_CMD: bool_ok,
+            CREATE_CMD: bool_ok,
             CREATERULE_CMD: bool_ok,
             DELETERULE_CMD: bool_ok,
         }
 
         RESP2_MODULE_CALLBACKS = {
```

Comment:
_RESP2
_RESP3

we don't want these in the public interface (throughout modules)